test/e2e/sourcery.js

```diff
@@ -5,45 +5,90 @@ module.exports = {
         browser
         .url(browser.launchUrl)
         .waitForElementVisible('h1'); // wait for the Login title
-        //Click to expand select user dropdown
+        //Click to expand select user dropdown--Step 1
         browser.element('css selector', '#react-select-2--value', function(result) {
             if(result.status != -1) { 
                 browser.click('#react-select-2--value');
             }
         });
-        //Select from expanded droprown
+       // browser.pause(10000);
+        //Select from expanded dropdown--Step 2
         browser.element('css selector', '[aria-label="Demo User"]', function(result) {
```

Comment:
You should use your user, not a "Demo User"

Reply:
In Vilnius Sourcebooks there is no user with my name, so I'm using Demo user